js/index.js

```diff
@@ -64,4 +66,48 @@ const messageList = messageSection.querySelector('ul');
     messageList.appendChild(newMessage);
     messageSection.hidden = false;
     messageForm.item(0).reset();
-});
\ No newline at end of file
+});
+
+const githubRequest = new XMLHttpRequest();
+githubRequest.open('GET', 'https://api.github.com/users/raissanfg/repos');
+githubRequest.send();
+githubRequest.addEventListener('load', function() {
+const repositories = JSON.parse(this.response)
+
+const projectSection = document.querySelector('#projects')
+const projectList = projectSection.querySelector('ul')
+
+ for (let i=0; i < repositories.length; i++) {
+    var project = document.createElement('li');
+    var repositoryLink = document.createElement('a');
+    repositoryLink.href = repositories[i].html_url;
+    repositoryLink.innerHTML = repositories[i].name;
+    projectList.appendChild(project);
+    project.appendChild(repositoryLink);
+
+ }
+});
```

Comment:
This looks good and looks like it's working correctly now 👍 

Since we only need one part of the code fetching repositories; you could comment this out now and just have the fetch code from the last lesson (lesson 6-2) do this work. In the end you won't need both but the exercise was to ensure you had practice with calling an API with both methods.

Reply:
The only problem I'm having, is that when I comment this out,  the list disappear from the page